src/common/components/Button/index.jsx

```diff
@@ -0,0 +1 @@
+export { default as MoreButton } from './MoreButton'
```

Comment:
What is the purpose of the `Button` file? What is the advantage over importing things from `MoreButton`?

Reply:
Didn't see this...
Is that kind of weird? I mean to pass a variable without changing anything, is that a complexity or an abstraction?

Reply:
My goal was to centralize in the `Button` "namespace" all things button-related (and these things will probably be integrated in cozy-ui one day, when it's ready), so you could do something like:
```js
import Button, { MoreButton, UploadButton } from 'common/components/Button'
```
For now, I just wanted to commonalize `MoreButton`, but my goal is to push here all our buttons and to evolve into more easy-to-use `Button` components.

Reply:
I must say I'm not too overwhelmed by this. The eventual idea is good, but we don't know when we'll add another button to this list, so I find it premature. But I can live with it.